VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -639,43 +669,40 @@ public void onFail(Exception ex) {
         }
     };
 
-    /**
-     * mostly the orientation changes.
-     * @param newConfig
-     */
-    @Override
-    public void onConfigurationChanged(Configuration newConfig) {
-        super.onConfigurationChanged(newConfig);
+    private void updateUIForOrientation(boolean isLandscape) {
```

Comment:
Let's remove the boolean argument here since we can just call `getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE`